<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DheenatalansAir.fios-router.home" failures="6" tests="47" name="runner.testrunner1" time="87.018" errors="0" timestamp="2024-09-30T13:18:51 EDT" skipped="0">
  <testcase classname="runner.testrunner1" name="runScenario" time="2.459"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.698"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.854"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.511"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.598">
    <failure message="The following asserts failed:
	Failure in scenario: empty firstName | Expected status code: 400 but got: 404" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty firstName | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByFirstName.validate_user_and_user_validations_for_get_request_as(getUserByFirstName.java:164)
at ✽.Validate "400" and user validations for get request as "empty firstName"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/src/test/resources/features/06_getUserByFirstName.feature:17)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: empty firstName | Expected status code: 400 but got: 404]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="0.178"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.510"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.523"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.513"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.159"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.525"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.653"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.613"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.613"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.538"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.549">
    <failure message="The following asserts failed:
	Failure in scenario: empty userid | Expected status code: 400 but got: 404" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty userid | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.deletUsertest.validate_and_other_data_validations(deletUsertest.java:168)
at ✽.Validate "400" and other data validations for scenario name as "empty userid"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/src/test/resources/features/07_deleteUserByID.feature:17)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: empty userid | Expected status code: 400 but got: 404]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="0.717"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.591">
    <failure message="The following asserts failed:
	Failure in scenario: empty userid | Expected status code: 400 but got: 404" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: empty userid | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByIDtest.validate_user_and_other_validations_for_scenario_name_as(getUserByIDtest.java:165)
at ✽.Validate "400" and other user validations for scenario name as "empty userid"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/src/test/resources/features/04_getUserByID.feature:17)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: empty userid | Expected status code: 400 but got: 404]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="1.795"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.517"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.901"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.543"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.236"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.589"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.693"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="6.212"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.575"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="1.360"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="26.325">
    <failure message="The following asserts failed:
	Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
	Failure in scenario: zipcode as empty | Expected status code: 400 but got: 201,
	Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 201,
Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.createUsertest.tearDown(createUsertest.java:231)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 201
Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="0.578"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.524"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.560"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.509">
    <failure message="The following asserts failed:
	Failure in scenario: firstName in integer | Expected status code: 400 but got: 404" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: firstName in integer | Expected status code: 400 but got: 404
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.getUserByFirstName.validate_user_and_user_validations_for_get_request_as(getUserByFirstName.java:164)
at ✽.Validate "400" and user validations for get request as "firstName in integer"(file:///Users/mathumathibalakrishnan/git/userAPI_RestAssured/src/test/resources/features/06_getUserByFirstName.feature:17)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: firstName in integer | Expected status code: 400 but got: 404]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="0.573"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="1.385"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.845"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.630"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.721"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.602"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.496"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.533"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.491"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.204"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.628"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="25.085">
    <failure message="The following asserts failed:
	Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
	Failure in scenario: invalid endpoint | Expected status code: 404 but got: 500 | Error Status: 500,
	Failure in scenario: null endpoint | Expected status code: 404 but got: 500 | Error Status: 500,
	Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
	Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500,
	Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200,
	Failure in scenario: contentType | Expected status code: 415 but got: 200" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only,
Failure in scenario: invalid endpoint | Expected status code: 404 but got: 500 | Error Status: 500,
Failure in scenario: null endpoint | Expected status code: 404 but got: 500 | Error Status: 500,
Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user,
Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500,
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200,
Failure in scenario: contentType | Expected status code: 415 but got: 200
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at stepdefinition.updateUsertest.tearDown(updateUsertest.java:305)
]]>
    </failure>
    <system-out>
      <![CDATA[Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only
Failure in scenario: invalid endpoint | Expected status code: 404 but got: 500 | Error Status: 500
Failure in scenario: null endpoint | Expected status code: 404 but got: 500 | Error Status: 500
Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user
Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200
Failure in scenario: contentType | Expected status code: 415 but got: 200]]>
    </system-out>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testrunner1" name="runScenario" time="0.532"/>
  <testcase classname="runner.testrunner1" name="runScenario" time="0.574"/>
  <system-out>
    <![CDATA[Failure in scenario: valid mandatory | Expected status code: 201 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 201
Failure in scenario: contentType | Expected status code: 415 but got: 409 | Error Status: 409 CONFLICT | Error Message: User already exist with same email id
Failure in scenario: empty userid | Expected status code: 400 but got: 404
Failure in scenario: valid mandatory | Expected status code: 200 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Plot number should contain alphaNumeric values only
Failure in scenario: invalid endpoint | Expected status code: 404 but got: 500 | Error Status: 500
Failure in scenario: null endpoint | Expected status code: 404 but got: 500 | Error Status: 500
Failure in scenario: phoneNumber as duplicate | Expected status code: 409 but got: 400 | Error Status: 400 BAD_REQUEST | Error Message: Contact number is already in use by another user
Failure in scenario: email as duplicate | Expected status code: 409 but got: 500 | Error Status: 500
Failure in scenario: zipcode as empty | Expected status code: 400 but got: 200
Failure in scenario: contentType | Expected status code: 415 but got: 200
Failure in scenario: firstName in integer | Expected status code: 400 but got: 404
Failure in scenario: empty firstName | Expected status code: 400 but got: 404
Failure in scenario: empty userid | Expected status code: 400 but got: 404]]>
  </system-out>
</testsuite> <!-- runner.testrunner1 -->
